massinstall/massinstall.go

```diff
@@ -125,10 +125,40 @@ func (mi *MassInstall) MustRun(args *args.Args) bool {
 func (mi *MassInstall) Run(md *model.SystemInstall, rootDir string, options args.Args) (bool, error) {
 	var instError error
 
-	// Need to ensure the partitioner knows we are running from
-	// the command line and will be using the whole disk
-	for _, curr := range md.TargetMedias {
-		md.InstallSelected[curr.Name] = storage.InstallTarget{WholeDisk: true}
+	// If there are no media defined, then we should look for
```

Comment:
dont strongly feel about it..but moving it a new function called is_advanced_partitioning_mode and using the return boolean to detect advanced mode or not will keep the run itself succinct. 

Upto you.

Reply:
Hmm...I see what you are getting at.
But we don't really know if it is advanced mode yet, we will just check if the advanced mode labels are used only if there are no explicit media defined in the YAML file.